### PR TITLE
Sample yaml CI platform, name and desciptions align

### PR DESCRIPTION
**Pull request by KAGA164**
This PR provides a valid setup platform for CI and also alignment for name and description in
Bluetooth and NFC samples

### CODE REVIEW

**Comment by KAGA164** [comment]
@grochu @jorgenmk Could you review ?

File: samples/bluetooth/central_bas/sample.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 sample:
-  description: TBD
-  name: TBD
+  name: BLE Central BAS
+  description: The bluetooth low energy central battery service application
```
Comment (grochu) [inline samples/bluetooth/central_bas/sample.yaml]: Remove "The" and use capital letters: _Bluetooth Low Energy_ for consistency with other descriptions.

**Comment by jorgenmk** [comment]
LGTM

**Comment by joerchan** [comment]
@KAGA164 Can you do the capital letter and sample update as suggested by grochu?

File: samples/bluetooth/central_bas/sample.yaml (hunk 1 above)
Comment (KAGA164) [reply samples/bluetooth/central_bas/sample.yaml]: Ok removed

**Comment by KAGA164** [comment]
> 
> 
> @KAGA164 Can you do the capital letter and sample update as suggested by grochu?

Done

**Pull request merged**